index.bs

```diff
@@ -1722,7 +1775,8 @@ WebAuthn supports multiple attestation types:
 
 ### Generating an Attestation Object ### {#generating-an-attestation-object}
 
-This section specifies the algorithm for generating an [=attestation object=] for any [=attestation statement format=].
+This section specifies the algorithm for generating an [=attestation object=] (see: [Figure 3](#fig-attStructs)) for any
```

Comment:
Including a literal figure number "Figure 3" is likely to cause numbering problems later.  If possible, please have this number be automatically generated by the tool, rather than included literally.